analysisstore/client/commands.py

```diff
@@ -10,10 +10,9 @@
 
 
 class AnalysisClient:
-    """Your path to AnalysisStore(yiiiiissssss, camelcase"""
+    """Client used to pass messages between analysisstore server and apps"""
     def __init__(self, host='localhost:8999'):
```

Comment:
I did smth like this for the archiver a while back. It doesn't get too complex, worth the effort.